editor/editor_inspector.cpp

```diff
@@ -382,12 +382,24 @@ void EditorProperty::update_property() {
 	GDVIRTUAL_CALL(_update_property);
 }
 
+void EditorProperty::_set_read_only(bool p_read_only) {
```

Comment:
I think this is pointless. I believe it might make sense to have this as the base implementation (in the EditorProperty class itself), but there no point in having it in all children classes. You should simply override `set_read_only(bool p_read_only)` on the children classes. 

Most functions in the engine don't check for scripts for performance reason, we only expose the ones that have a clear use case.

Reply:
Does this mean that these are meaningless?
https://github.com/godotengine/godot/pull/51722#issuecomment-903175425

Reply:
I guess. I think allowing custom behavior makes sense for the base class (most, if not all, scripts will inherit from EditorProperty directly) but not for the children one.